frontend/src/app/main/ui/workspace/sidebar/shortcuts.cljs

```diff
@@ -0,0 +1,294 @@
+  [{:keys [content command] :as props}]
+  (let [managed-list (if (coll? content)
+                       content
+                       (conj () content))
+        split-sc     (fn [sc]
+                       (let [sc (cond-> sc (str/includes? sc "++")
+                                        (str/replace "++" "+plus"))]
+                         (if (= (count sc) 1)
+                           [sc]
+                           (str/split sc #"\+| "))))
+        keys-list    (map split-sc managed-list)]
+    [:span.keys
+     (for [keys keys-list]
+       (let [last-element (last keys-list)]
+         [:*
+          (for [key keys]
+            (let [modified-keys {:up    ds/up-arrow
+                                 :down  ds/down-arrow
+                                 :left  ds/left-arrow
+                                 :right ds/right-arrow
+                                 :plus "+"}
+                  macos-keys    {:command "\u2318"
+                                 :option  "\u2325"
+                                 :alt     "\u2325"
+                                 :delete  "\u232B"
+                                 :del     "\u232B"
+                                 :shift   "\u21E7"
+                                 :control "\u2303"
+                                 :esc     "\u238B"
+                                 :enter   "\u23CE"}
+                  is-macos?     (cf/check-platform? :macos)
+                  key           (if (contains? modified-keys (keyword key)) ((keyword key) modified-keys) key)
+                  key           (if (and is-macos? (contains? macos-keys (keyword key))) ((keyword key) macos-keys) key)
+                  unique-key    (str (d/name command) "-" key)]
+              [:*
+               [:span.key-box {:key unique-key} key]]))
+          (when (not= keys last-element) [:span.space ","])]))]))
+
+(mf/defc shotcut-row-generator
+  [{:keys [shortcuts elements filter-term match-section? match-subsection?] :as props}]
+  (let [reduce-translation (fn [acc element] (assoc acc element (tr (str "shortcuts." (d/name element)))))
+        translations       (reduce reduce-translation {} elements)
+        filtered           (if (or match-section? match-subsection?)
+                             (vals translations)
+                             (filter #(matches-search % (:term @filter-term)) (vals translations)))
+        sorted-filtered    (sort filtered)]
+    [:ul.sub-menu
+     (for [command-translate sorted-filtered]
+       (let [command (first (filter (comp #{command-translate} translations) (keys translations)))
+             content (:command (command shortcuts))
+             name    (tr (str "shortcuts."  (d/name command)))]
+         [:li.shortcut-name {:key command}
+          [:span.command-name name]
+          [:& shortcuts-keys {:content content
+                              :command command}]]))]))
+
+(mf/defc section-title
+  [{:keys [is-visible? section elem-n] :as props}]
```

Comment:
Perhaps this number should reflect the count of filtered elements, when a search is active? I think it could be done moving the filter to the upper component that has the section-title, and pass the already filtered list to the child component.

Reply:
It's a very good idea¡¡